README.md

```diff
@@ -1 +1 @@
-# Microverse-start-project
\ No newline at end of file
+# Microverse-start-project
```

Comment:
- [ ] Kindly modify your README file, please use this [template](https://github.com/microverseinc/readme-template) to copy all the sections and modify the placeholder text with information about your project for a professional documentation 👍